complete_python_pro_bootcamp/Day 14 Higher or lower followers game/Higher or Lower Project/game_data.py

```diff
@@ -0,0 +1,302 @@
+data = [
+    {
+        'name': 'Instagram',
+        'follower_count': 346,
+        'description': 'Social media platform',
+        'country': 'United States'
```

Comment:
Please add the country flag.

For example:
        'name': 'Instagram',
        'follower_count': 346,
        'description': 'Social media platform',
        'country': 'United States'
         'country_flag': '🇺🇸',